tests/test_visitors/test_ast/test_classes/test_base_classes/test_base_builtin_classes.py

```diff
@@ -12,6 +12,13 @@
 enum_with_primitive3 = 'class First({0}, EnumMeta): ...'
 enum_with_primitive4 = 'class First({0}, enum.EnumType): ...'
 
+repr_enum_with_primitive1 = 'class First({0}, enum.StrEnum): ...'
```

Comment:
Maybe let's use some other name? `ReprEnum` can be subclassed with a primitive. But, `IntEnum` can't.

Maybe something like `concrete_enum`?

Reply:
```python
>>> import enum
>>> class A(int, enum.ReprEnum):
...     x = 1
...     y = 2
...     
>>> A.x, A.y
(<A.x: 1>, <A.y: 2>)
```